managed/cmd/pmm-managed-init/main.go

```diff
@@ -34,6 +35,12 @@ func main() {
 	if on, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); on {
 		logrus.SetLevel(logrus.TraceLevel)
 	}
+	if ok, _ := strconv.ParseBool(os.Getenv("DISABLE_INTERNAL_DB")); ok {
```

Comment:
```suggestion
	if ok, _ := strconv.ParseBool(os.Getenv("PMM_DISABLE_INTERNAL_DB")); ok {
```
Can we start applying a `PMM_` prefix to all variables? I provided some background in [this doc](https://confluence.percona.com/display/PMM/PMM+v3.0) (section 3).